lib/translation_maps/additional_country_names_to_geocode_ids.yaml

```diff
@@ -1,4 +1,5 @@
 peoples r china: 1814991
+mongol peo rep: 2029969
```

Comment:
Should we alphabetize by country? Expect this list to grow.

Reply:
🙄